sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/AssetLengthUnit.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> Represents the measurement of a single distance unit in an asset. </summary>
+    public enum AssetLengthUnit
```

Comment:
Can you remind me why this isn't generated?

Reply:
This is a helper for generating a scale float, it's never actually referenced or used in the Swagger. All client side magic!

Reply:
That doesn't make sense to me.